backend/db/ddl-14-concepts_with_counts.jinja.sql

```diff
@@ -19,6 +19,3 @@ CREATE TABLE IF NOT EXISTS {{schema}}concepts_with_counts{{optional_suffix}} AS
     ORDER BY concept_id, domain );
 
 CREATE INDEX cc_idx1{{optional_index_suffix}} ON {{schema}}concepts_with_counts{{optional_suffix}}(concept_id);
-
--- the following drop table is causing errors with the initialize script
--- DROP TABLE {{schema}}concepts_with_counts_ungrouped{{optional_suffix}} CASCADE;
```

Comment:
## Removed comment
We have diff DDLs for `concepts_with_counts_ungrouped` and `concepts_with_counts`, so I don't think that dropping the former is something we want to do. It's currently a table that we have, and I think we're using. If we ever do want to drop it during creation of `concepts_with_counts`, then we should make a single DDL file for the both of them.